tox.ini

```
@@ -168,3 +170,16 @@ passenv =
 skip_install = false
 allowlist_externals =
   git
+
+[testenv:changelog-draft]
+deps =
+    towncrier
+    -r{toxinidir}/docs/requirements.txt
```

Is this really needed to just call Towncrier? Or are you using this as a hack for "constraints"? In that case, we should do
```suggestion
    -c{toxinidir}/docs/requirements.txt
```
instead (it won't explode because there's no hashes).

---

I wanted to pin the `towncrier` version to match the doc build.

> it won't explode because there's no hashes

Ah! Right, okay, I'll switch it to `-c`.